### ISSUE TITLE
South Sudan doesn't exist in country dropdown

### ISSUE THREAD

**Issue by mclapham** [paleobiodb/classic#121]
Fairly self-explanatory from the title! I found a reference with fossils from South Sudan (in an attempt to get fossils from every country) but "South Sudan" isn't in the country dropdown.

**Comment by markuhen** [comment]
It would be great if we could switch this over to a data service vs. a static list.

**Comment by jpjenk** [comment]
Yes that is a good idea. The allowable country list appears in three different locations within the classic code and classic templates but it is not difficult to manually update periodically.

Building data service calls into the classic codebase, while possible, could fall into the same category of selective hybridization that was discussed when reviewing the taxon search form. Perhaps the best way to do this would be an AJAX call from the browser code (which is itself deemed legacy at this point) that could populate the dropdown with the latest country list.
